### ISSUE TITLE
Develop a product page

### ISSUE THREAD

**Issue by tpendragon** [projecthydra/curation_concerns#920]
Look towards things like http://spotlight.projectblacklight.org/ and http://geoconcerns.github.io/.

If we're going to be a core piece of infrastructure, we should present it as such and validate the usefulness of this layer.

**Comment by mjgiarlo** [comment]
Yes, great idea. Part of why we created http://sufia.io/ is because lots of folks had questions that weren't easily answered on the Hydra wiki or GitHub wikis, so I suspect this would be useful for CurationConcerns as well. If it makes sense to for Sufia and CC to share a site (esp. given some of the Hydra AWG conversations about their future trajectories) I'd be happy to make that happen. Else, it's :100: OK for them to have their own domains and identities and am happy to help with that too.
